js/homeview.js

```diff
@@ -743,7 +743,7 @@ enyo.kind({
 		this.$.neighborbutton.setNodeProperty("title", l10n.get("NeighborhoodView"));
 		this.$.helpbutton.setNodeProperty("title", l10n.get("Tutorial"));
 		if (app.localize) {
-			app.localize();
+			app.localize;
```

Comment:
Very strange fix: removing parenthesis is equivalent to don't call the function at all.

Reply:
Restored parenthesis